src/runner.ts

```diff
@@ -225,13 +225,17 @@ export class ActionRunner {
           usersToAdd = [...new Set([...usersToAdd, ...members])];
         }
       }
-      requirements.push({ users: usersToAdd, requiredApprovals: reviewers.min_approvals });
+      requirements.push({
+        users: usersToAdd,
+        requiredApprovals: reviewers.min_approvals,
+        countAuthor: reviewers.countAuthor ?? false,
+      });
     }
 
     // We count how many reviews are needed in total
     const requiredAmountOfReviews = rule.reviewers.map((r) => r.min_approvals).reduce((a, b) => a + b, 0);
     // We get the list of users that approved the PR
-    const approvals = await this.prApi.listApprovedReviewsAuthors();
+    const approvals = await this.prApi.listApprovedReviewsAuthors(false);
```

Comment:
Actually why don't we do this here?

```suggestion
    const approvals = await this.prApi.listApprovedReviewsAuthors(rule.countAuthor ?? false);
```

Reply:
It's not possible I'm afraid, as rule doesn't have the `countAuthor` property.

It is nested as `rule.reviewers[].countAuthor` so I need to extract it during that iteration.

Reply:
Ahh right, I get it now.

Reply:
`rule.reviewers[].countAuthor` has been moved to `rule.countAuthor` so now it is possible 😄